.github/workflows/node.yml

```diff
@@ -9,10 +9,9 @@ on:
         types: [published]
 
 jobs:
-    build:
+    test:
         runs-on: ubuntu-latest
-
-        container: node:20-alpine
+        container: node:20-slim
```

Comment:
tar included in alpine is not standard and causes errors with setup-node action.